### DISCUSSION TITLE
PyVista integration with Plotly

### DISCUSSION THREAD

**Discussion by drakegeo** [pyvista/pyvista#3670] [Q&A]
Hi all,

I am implementing parametric numerical simulation with machine learning, and I was wondering if there is any way to couple the plotly (link: https://plotly.com/) with PyVista?

**Comment by banesullivan** [comment]
Take a look at https://dash.plotly.com/vtk

They have some demos "converting" pyvista to Plotly/Dash: https://dash.plotly.com/vtk/advanced
